web/src/elements/table/TableSearch.ts

```diff
@@ -41,23 +76,24 @@ export class TableSearch extends AKElement {
             @submit=${(e: Event) => {
                 e.preventDefault();
                 if (!this.onSearch) return;
-                const el = this.shadowRoot?.querySelector<HTMLInputElement>("input[type=search]");
+                const el = this.shadowRoot?.querySelector<HTMLInputElement>("[name=search]");
```

Comment:
```suggestion
                const el = this.shadowRoot?.querySelector<HTMLTextAreaElement>("[name=search]");
```
as per the change below, I think